src/model/encoding/__tests__/convertFromHTMLToContentBlocks-test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails isaac, oncall+ui_infra
+ */
+
+'use strict';
+
+jest.disableAutomock();
+
+const convertFromHTMLToContentBlocks = require('convertFromHTMLToContentBlocks');
+const DefaultDraftBlockRenderMap = require('DefaultDraftBlockRenderMap');
+
+function testConvertingAdjacentHtmlElementsToContentBlocks(
+  tag: string
+) {
+  it(`must not merge tags when converting adjacent <${tag} />`, () => {
+    const html_string = `
+      <${tag}>a</${tag}>
+      <${tag}>b</${tag}>
+    `;
+
+    const blocks = convertFromHTMLToContentBlocks(html_string);
+
+    expect(blocks.contentBlocks.length).toBe(2);
+  });
+}
+
+function performTestWithDefaultDraftBlockTags(
```

Comment:
This test is pretty hard to read. It would probably be clearer if you just hardcoded a couple HTML examples.